src/async-profiler/cgotraceback.cpp

```diff
@@ -0,0 +1,223 @@
+#include <cstring>
```

Comment:
I think you can delete this file and cgotraceback.go, they're specifically for my Go library :)